firefox-ios/firefox-ios-tests/Tests/XCUITests/SearchTest.swift

```diff
@@ -280,4 +280,12 @@ class SearchTests: BaseTestCase {
         waitForTabsButton()
         validateSearchSuggestionText(typeText: "localhost")
     }
+    
+    func testFirefoxSuggestionsSettings(){
```

Comment:
I know this probably isn't the most awesome test, and we should probably test the actual functionality of seeing a FX sponsored suggestion, but I wasn't sure how to go about that. Particularly, do I choose a sponsored suggestion keyword that currently exists, that may not be in the future so therefore it could be flaky? or do I mock a returned sponsored suggestion? or am i off base and I should do something else entirely? Was hoping for some guidance 🙂